pkg/model/dev.go

```diff
@@ -832,7 +800,7 @@ func (dev *Dev) LabelsSelector() string {
 }
 
 // ToTranslationRule translates a dev struct into a translation rule
-func (dev *Dev) ToTranslationRule(main *Dev, reset bool) *TranslationRule {
+func (dev *Dev) ToTranslationRule(namespace string, main *Dev, reset bool) *TranslationRule {
```

Comment:
NIT: In Go, variables are ordered by importance, so I think it's weird to have the namespace preceding the dev. During the code we always set `*Dev` first, but this one is just a NIT

Reply:
good catch, I've changed all the others to have dev first but not sure what happened here. Fixed in bc31d56ec5ec18dfe0f6ab88d34bbfb044847eb4